### ISSUE TITLE
Schema constants

### ISSUE THREAD

**Issue by shannonwells** [LibertyDSNP/mrc#70]
Determine min and max schema size, max schema registrations

* Min Schema size: it's an easier check to bail up front on a minimum size and we should. 
* Max schema size: determined by max weight and also @enddynayn suggests this can be limited before the message even gets put in the mempool by restricting the weight for the extrinsic.
* Max Schema registrations:  there was some debate about whether there should be a number below whatever the max of a U16 is.  If the limit is just U16 max value then it would seem we don't need this config?  Perhaps it would be better to leave some wiggle room though.

**Comment by saraswatpuneet** [comment]
@wilwade  @shannonwells  @enddynayn  not sure if there is a bug, But currently in runtime
the max is set to 
	type SchemaModelMaxBytesBoundedVecLimit = ConstU32<65_000>;

while 
	pub const MaxMessagePayloadSizeBytes: u32 = 1024 * 50; // 50K

I think we got confused with this 1024, which states max message size per schema about 50k

We need to change SchemaModelMaxBytesBoundedVecLimit to 1024 ?

**Comment by saraswatpuneet** [comment]
Yep just checked, max schema size is tied to max weight so all good :+1:
